README.md

```diff
@@ -1,26 +1,78 @@
+<!-- markdownlint-disable MD026 MD033 -->
+
 # DiscordFeedback
-The bot that powers the Discord Feedback server
 
-# Selfhosting
-We don't (fully) support hosting your own copy of DiscordFeedback, this program is highly customized and not suited to run outside of the official Discord Feedback server.
+This is the bot that powers the Discord Feedback server, develop'd by the volunteer group only known as "Discord Feedback Bot Tinkerers Sqaud™"
+
+## Table of Contents
+
+- [Development](#development)
+  - [Config](#config)
+  - [Global ESlint Install](#global-eslint-instal)
+  - [Testing](#coming-soon)
+- [Selfhosting](#selfhosting)
+  - [Config Creation](#config-creation)
+  - [Database Initalization](#database-initialization)
+  - [Running the Bot](#running-the-bot)
+
+## Development
+
+We here at Discord Feedback Bot Tinkerers Sqaud™ use Visual Studio Code, with the [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint), [npm](https://marketplace.visualstudio.com/items?itemName=eg2.vscode-npm-script), [npm Intellisense](https://marketplace.visualstudio.com/items?itemName=christian-kohler.npm-intellisense) plugins in order to produce the bestest code.
+
+### Config
+
+We like to keep things simple, when working on the bot just add ``debug: true,`` to the config, right after the first line
```

Comment:
This is going to be outdated pretty quickly

Reply:
We planning on not supporting a development mode soon?

Reply:
`debug: true` is going to be deprecated once the EXP update is out.

Reply:
Then how will we test commands against the staging instance? or are you switching to a different way to handle that?

Reply:
It's going to change entirely

Reply:
Want to share once its done?